osc-check_source.py

```diff
@@ -257,6 +257,7 @@ def _checker_one_request(self, rq, opts):
             self._devel_projects['filesystems:ceph/'] = 'ceph-deploy'
             self._devel_projects['Virtualization:Appliances:Builder/'] = 'python3-kiwi'
             self._devel_projects['isv:ownCloud:desktop/'] = 'owncloud-client'
+            self._devel_projects['system:snappy/'] = 'snappy'
```

Comment:
Or cleanup again - in the end we only need the list temporarily until at least one package has the devel prj set to anything there.... splitting that out in a differently maintained format sounds like a good idea indeed (but should not block moving forward in the time being)